src/Selenium2Driver.php

```diff
@@ -863,6 +863,21 @@ public function keyUp($xpath, $char, $modifier = null)
     }
 
     /**
+     * Simulates keyboard typing with Syn.type().
```

Comment:
Mink drivers are abstractions. So the phpdoc of methods should absolutely not describe them by talking about their internal implementation.

and indeed, as @aik099 said, this method won't be accepted if it is not properly added to the Mink API (btw, people are not expected to use the driver directly)